concourse/templates/default.mako

```diff
@@ -29,10 +30,10 @@ default_container_registry = config_set.container_registry()
 # expose secrets_server endpoint to all jobs
 secrets_server_cfg = config_set.secrets_server()
 
-if secret_cfg:
-  secrets_server_cc_cfg_name = secrets_server_cfg.secrets().encrypted_concourse_cfg_name()
-else:
-  secrets_server_cc_cfg_name = secrets_server_cfg.secrets().concourse_cfg_name()
+secrets_server_cc_cfg_name = secrets_server_cfg.secrets().secret_url_path(
```

Comment:
late addition to the review, but I think somewhat related to my confusion:

`secrets_server_cc_cfg_name = secrets_server_cfg.secrets().secret_url_path(…)`

Is the config name actually a url? Also, is it so by design?

Reply:
it's not a fully qualified url but a path to the secret

Reply:
@AndreasBurger : `URL` would be the fully qualified endpoint (<schema>://<netloc>/<path>[<query>}), whereas url_path was intended to only refer to the `<path>` part